dtcpcbios/video.src

```diff
@@ -313,7 +313,7 @@ SetModeJump1:
 
                 Xor     Bh,Bh                   ;Form index into table (Bl=mode)
                 Push    Bx                      ;Save Bx=crt mode
-                Mov     Bl,OffsetTable[Bx]      ;Load the offset
+                Mov     Bl,byte ptr Cs:OffsetTable[Bx]      ;Load the offset
```

Comment:
Not sure about this one. I know it's required for building. In fact, I had to do something in this very same line for MASM. However, if the idea is to keep this copy as original as possible. What do you think?

Reply:
I agree that this one may not be for the "time capsule" version. It seems like the result of either a quirk/bug in ASM86 1.x, or a missing directive somewhere for the segment override. The `Cs:` is or course needed and present in the assembled binary of the original listing, but absent in the assembly source. Perhaps once we can find a copy of asm86 1.x we can verify this.

I'll update the PR and back it out.

Reply:
Sounds good! I'll be waiting for the change. Thanks a lot!